tests/benchmarks/fio_benchmark_test.py

```diff
@@ -153,5 +154,62 @@ def TestZeroMinutes(self):
     self.assertEquals(proc.call_count, 0)
 
 
+class TestFioTargetModeFlag(unittest.TestCase):
+  def doTargetModeTest(self, mode, fill_device=None, against_device=None,
+                       format_disk=None):
+    with mock.patch(fio_benchmark.__name__ + '.FillDevice') as fill_device, \
```

Comment:
Done. I kept the `fio_` prefix on FLAGS because there are many different variables named FLAGS in different modules and I wanted to be clear which I was changing.